app/components/post_draft/post_draft.js

```diff
@@ -131,6 +141,97 @@ export default class PostDraft extends PureComponent {
         return messageLength > 0;
     };
 
+    showSendToGroupsAlert = (groupMentions, memberNotifyCount, channelTimezoneCount) => {
```

Comment:
Would love to see this as a utility or helper function outside the component, for readability. Maybe do the same for other similar functions in this component.

Reply:
Can a ticket be made for refactoring this component separately and addressed at another time?

Reply:
A ticket has been made to keep track of this refactoring as per our discussions.
https://mattermost.atlassian.net/browse/MM-26876